src/common/environment-helper.ts

```diff
@@ -0,0 +1,17 @@
+// production environment= "production"
+// local development = "development"
+// preview release = "staging"
+// feature branch (github PRs)= "feature"
+// test environment = "test"
+
+import { IS_TEST_ENV } from './constants';
+
+function getEnvironment() {
+  if (IS_TEST_ENV) return 'test';
+  if (process.env.NODE_ENV === 'development') return 'development';
+  if (!process.env.PREVIEW_RELEASE && !process.env.IS_PUBLISHING) return 'feature';
+  if (process.env.PREVIEW_RELEASE) return 'staging';
+  return 'production';
```

Comment:
See https://github.com/hirosystems/stacks-wallet-web/pull/1984, should make detection of environment easier.

Would also be cool to have a `whenEnvironment` helper, ala `whenChainId`,

```ts

type WalletEnvironments = 'production' | 'preview' | 'feature' | 'development';
type WhenEnvironmentMap<T> = Record<WalletEnvironments, T>;
export function whenEnvironment<T>(envMap: WhenEnvironmentMap<T>) {
  return envMap[process.env.WALLET_ENVIRONMENT as WalletEnvironments];
}
```